### ISSUE TITLE
Escalation of privileges

### ISSUE THREAD

**Issue by d0ubl3puls4r** [r00t-3xp10it/meterpeter#2]
Hello, I would like to know how it works to escalate privileges in the system, I even went to the postexploit->escalate->getsystem options

and it looks like the process was OK, but how to access the session with those elevated privileges, from now on I don't know what else to do.
Thanks in advance.

**Comment by r00t-3xp10it** [comment]
This WIKI explains all the steps required to elevate current session ...
https://github.com/r00t-3xp10it/meterpeter/wiki/WSReset.exe-Privilege-Escalation-(Client.ps1)

**Comment by d0ubl3puls4r** [comment]
Thanks, I managed to understand and it worked perfectly here, now my question is about the persistent mode, I activated the option, I left the program, restarted the windows machine and I was unable to return to the system access.

**Comment by r00t-3xp10it** [comment]
what **persistence** did you have pick up ??
meterpeter as 5 available persistence mechanisms ...

**Comment by d0ubl3puls4r** [comment]
> what **persistence** did you have pick up ??
> meterpeter as 5 available persistence mechanisms ...
> 
> **`Remark:`** persistence does not give you **SYSTEM** accesss by its own ..



> what **persistence** did you have pick up ??
> meterpeter as 5 available persistence mechanisms ...
> 
> **`Remark:`** persistence does not give you **SYSTEM** accesss by its own ..

I used almost all the options but I did not get a reverse access from the remote system, my question is whether the persistent mode serves as a Backdoor allowing to return to the system how does this access work?  does it happen reversely?  do I have to put the tool back into listening mode on the same IP and port?

**Comment by r00t-3xp10it** [comment]
yes... you have to put meterpeter in listenning mode to wait for connection ..
